### PR TITLE
add e2e test for stress

### PR DESCRIPTION
**Pull request by YangKeao**
Signed-off-by: Yang Keao <keao.yang@yahoo.com>

### What problem does this PR solve?

Add e2e test for `StressChaos`, so that I can do my refractor in #1578 more confidently.

### CODE REVIEW

File: test/e2e/chaos/stresschaos/cpu.go [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+		if err != nil {
+			return false, err
+		}
+
+		diff[0] = conditions[0].CpuTime - lastCPUTime[0]
+		diff[1] = conditions[1].CpuTime - lastCPUTime[1]
+		lastCPUTime[0] = conditions[0].CpuTime
+		lastCPUTime[1] = conditions[1].CpuTime
+		framework.Logf("get CPU: [%d, %d]", diff[0], diff[1])
+		if diff[0] > 5e8 && diff[1] < 5e6 {
```
Comment (WangXiangUSTC) [inline test/e2e/chaos/stresschaos/cpu.go:100]: how about adding a comment about why the value is `5e8` and `5e6`?

File: test/e2e/chaos/stresschaos/cpu.go [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+		if err != nil {
+			return false, err
+		}
+
+		diff[0] = conditions[0].CpuTime - lastCPUTime[0]
+		diff[1] = conditions[1].CpuTime - lastCPUTime[1]
+		lastCPUTime[0] = conditions[0].CpuTime
+		lastCPUTime[1] = conditions[1].CpuTime
+		framework.Logf("get CPU: [%d, %d]", diff[0], diff[1])
+		if diff[0] < 1e7 && diff[1] < 5e6 {
```
Comment (WangXiangUSTC) [inline test/e2e/chaos/stresschaos/cpu.go:124]: ditto

**Review by WangXiangUSTC — APPROVED** [review]
rest LGTM

File: test/e2e/chaos/stresschaos/cpu.go [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+	err := cli.Create(ctx, memoryStressChaos.DeepCopy())
+	framework.ExpectNoError(err, "create stresschaos error")
+
+	By("waiting for assertion some pods are experiencing memory stress ")
+	err = wait.Poll(time.Second, 15*time.Second, func() (done bool, err error) {
+		conditions, err := probeStressCondition(c, peers, ports)
+		if err != nil {
+			return false, err
+		}
+		if conditions[0].MemoryUsage > 50*1024*1024 && conditions[1].MemoryUsage < 50*1024*1024 {
```
Comment (AsterNighT) [inline test/e2e/chaos/stresschaos/cpu.go:47]: That's a bit counterintuitive to me. I wonder why is it not something like `conditions[0].MemoryUsage-conditions[1].MemoryUsage>50*1024*1024`?

**Review by AsterNighT — APPROVED** [review]
LGTM but for one thing.

File: test/e2e/chaos/stresschaos/cpu.go (hunk 3 above)
Comment (YangKeao) [reply test/e2e/chaos/stresschaos/cpu.go]: Fixed.

File: test/e2e/chaos/stresschaos/cpu.go (hunk 2 above)
Comment (YangKeao) [reply test/e2e/chaos/stresschaos/cpu.go:83]: Fixed

File: test/e2e/chaos/stresschaos/cpu.go (hunk 1 above)
Comment (YangKeao) [reply test/e2e/chaos/stresschaos/cpu.go:57]: Fixed

**Pull request merged**